### DISCUSSION TITLE
reduced Dynamic Orbits -  ACV

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#187] [Q&A]
Hello
 
Parisa has updated the ACVs in the kinematic orbit part of the processing (5 Step Process) for the CubeSats. If we want to now process the reduced dynamic orbits should we re-update the AVCs again in the reduced dynamic orbit processing? With the same 5 Step process? Or can we use the ones which we have already determined before (kinematic Orbits)?

Thank your for the feedback and the help

Cheers
Jean

**Comment by tmayerguerr** [comment]
Hi Jean,
ACV should be independent of the parametrization of the orbit. So you can use the already determined ones.
regards
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten

Thank you for the feedback and help

Cheers
Jean
